webpack.config.js

```diff
@@ -39,7 +39,8 @@ if (process.env.NODE_ENV === 'production') {
 			uglifyOptions: {
 				mangle: false,
 				output: {
-					beautify: true
+					beautify: true,
+					indent_level: 1 // eslint-disable-line camelcase
```

Comment:
Can you add a short comment on why it's there so we don't remove it in the future by accident?